rcsb/workflow/wuw/PdbCsmImageWorkflow.py

```diff
@@ -0,0 +1,86 @@
+"""Images workflow functions that actually do the work."""
+##
+# File: PdbCsmImageWorkflow.py
+# Date: 11-Dec-2024  mjt
+#
+#  Pdb and Csm image generation  - for workflow pipeline
+#
+#  Updates:
+#  12-dec-2024 mjt Created class object
+#
+##
+__docformat__ = "google en"
+__author__ = "Michael Trumbull"
+__email__ = "michael.trumbull@rcsb.org"
+__license__ = "Apache 2.0"
+
+from pathlib import Path
+import logging
+import subprocess
+import os
+
+
+logging.basicConfig(level=logging.INFO, format="%(asctime)s [%(levelname)s]-%(module)s.%(funcName)s: %(message)s")
+logger = logging.getLogger()
+
+
+class PdbCsmImageWorkflow:
+
+    def imagesGenJpgs(self, **kwargs: dict) -> None:
+        """Generate jpgs for given pdb/csm list."""
+        idListName = kwargs.get("idListName")
```

Comment:
Along with this kwarg-extracted variable, I would extract all possible kwargs up here and then reference the variables down below. That makes it clearer to the user (i.e., us) as to which keywords arguments are actually possible/available.

Also, when you extract them, be sure to assign a default value, such as  
```python

idListName = kwargs.get("idListName", None)
```